### PR TITLE
send-link-to-chat 1: components and storyshots

### PR DESCRIPTION
**Pull request by songgao**
I haven't done anything for mobile yet; will do it in a follow-up PR.

### CODE REVIEW

**Comment by songgao** [comment]
Hmm I'm realizing mobile is actually a lot different from desktop design. So will do that in a separate ticket, maybe after attachment flow is done too.

File: shared/fs/common/send-in-app-action.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import * as Styles from '../../styles'
+import {namedConnect} from '../../util/container'
+
+type OwnProps = {
+  path: Types.Path,
+  routePath?: ?I.List<string>,
+  sendIconClassName: string,
+}
+
+const mapDispatchToProps = (dispatch, {path, routePath}: OwnProps) => ({
```
Comment (chrisnojima) [inline shared/fs/common/send-in-app-action.js]: why is this connected

File: shared/fs/send-link-to-chat/index.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    case 'person':
+      return props.conversation.name
+    case 'group':
+      return 'your group'
+    case 'small-team':
+    case 'big-team':
+      return 'team members'
+    case 'none':
+      return ''
+    default:
```
Comment (chrisnojima) [inline shared/fs/send-link-to-chat/index.js]: See util/flow

File: shared/fs/common/send-in-app-action.js (hunk 1 above)
Comment (songgao) [reply shared/fs/common/send-in-app-action.js]: It needs to dispatch an action. This is populated in the next PR https://github.com/keybase/client/pull/15002

**Comment by songgao** [comment]
@chrisnojima PTAL thanks!

**Pull request merged**